static.config.js

```diff
@@ -374,6 +380,14 @@ const getGuidePageData = async (
 
   let guidePage = allGuidePages.edges[0].node;
 
+  // keeping this logic in there for now, stuff is kinda messy
+  let contacts = cleanContacts(guidePage.contacts);
+
+  // Guide pages only support single contacts for now
+  if (contacts && contacts.length) {
+    guidePage.contact = contacts[0];
+  }
```

Comment:
Does joplin prevent more than one contact for guide pages?

Reply:
no more so than we were already doing in the Guide page component before